code-challenges.test.js

```diff
@@ -17,78 +17,173 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-var num1 = 15
-// Expected output: "15 is divisible by three"
-
-var num2 = 0
-// Expected output: "0 is divisible by three"
-
-var num3 = -7
-// Expected output: "-7 is not divisble by three"
-
-
-
-
+// Name of the function will be 'divisibleByThree'
+describe("divisbleByThree", () => {
+    var num1 = 15
+    // Expected output: "15 is divisible by three"
+    it("return '15 is divisible by three' for an input of number 15", () => {
+        expect(divisibleByThree(num1)).toEqual('15 is divisible by three')
+    })
+
+    var num2 = 0
+    // Expected output: "0 is divisible by three"
+    it("return '0 is divisible by three' for an input of number 0", () => {
+        expect(divisibleByThree(num2)).toEqual('0 is divisible by three')
+    })
+
+    var num3 = -7
+    // Expected output: "-7 is not divisble by three"
+    it("return '-7 is not divisible by three' for an input of number -7", () => {
+        expect(divisibleByThree(num3)).toEqual('-7 is not divisible by three')
+    })
+})
 
 // b) Create the function that makes the test pass.
 
-
+// Create a function that takes in 1 argument that is a number
+const divisibleByThree = (number) => {
+    // The number will check if it is divisible by three by using the modulo
+    // and check that the remainder is zero
+    // Case 1 - it IS divisible by 3
+    if (number % 3 === 0) {
+        // Return a string saying the number is divisible by 3
+        return `${number} is divisible by three`;
+    // Case 2 - it is NOT divisible by 3
+    } else if (number % 3 !== 0) {
+        // Return a string saying the number is not divisible by 3
+        return `${number} is not divisible by three`;
+    }
+}
 
 
 
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-var randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
-
-var randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
-// Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
-
-
+// Name of function will be capitalizeFirstLetter
+describe("capitalizeFirstLetter", () => {
+    var randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+    var randomNouns1Expected = ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+    // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+    it('return ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"] for an input of ["streetlamp", "potato", "teeth", "conclusion", "nephew"]', () => {
```

Comment:
the problems are quite simple so it makes sense that descriptive assertions here involve stating expected inputs and outputs. Generally, you'll want to generalize the expectation a little so that it looks less like code and more like something you can tell someone in plain English. Something like, "returns a list of words with the first letter capitalized". That way we wouldn't have to make a test for each possible input.